pkg/nuclide-remote-connection/lib/RemoteFile.js

```diff
@@ -254,10 +254,11 @@ export class RemoteFile {
   }
 
   async read(flushCache?: boolean): Promise<string> {
-    const data = await this._getFileSystemService().readFile(this._localPath);
+    const encoding = this.getEncoding();
+    const options = encoding ? {encoding} : {};
```

Comment:
The Atom code always keeps this non-null/non-empty (and we've Flow-typed it as such), so I think that's fine to assume here as well. Let's just inline `options` as `{encoding}` below.